install/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
[This](https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script) SO post claims `set -e` is bad practice. Probably isn't in this script, but I thought I'd share

Reply:
We could (should?) have proper error handling, but it is better to stop the script than to continue and print "Successfully installed ..." at the end, when in  fact it failed.

Was not going to spend a lot of time to adress this, so went for a 'set -e' to at least improve the situation.